internal/utils/test/snapshot.go

```diff
@@ -0,0 +1,10 @@
+package test
+
+import (
+	"path"
+
+	"github.com/bradleyjkemp/cupaloy/v2"
+)
+
+var Snapshotting = cupaloy.New(
+	cupaloy.SnapshotSubdirectory(path.Join("testdata", "snapshots")))
```

Comment:
I think `path/filepath` is what we want to use here.

Reply:
Yeah, after reading up the documentation this should be the more stable choice.    
I have replaced all usages of `path` regarding file operations.